### PR TITLE
Add cfn-flip to conda-forge

### PR DESCRIPTION
**Pull request by rluria14**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/cfn-flip/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  commands:
+    - cfn-flip --help
+  requires:
+    - pytest >=4.3.0
+    - pytest-cov
+    - pytest-runner
+
+about:
+  home: https://github.com/awslabs/aws-cfn-template-flip
+  license: Apache2
```
Comment (marcelotrevisani) [inline recipes/cfn-flip/meta.yaml]: Could you please change it to
```suggestion
  license: Apache-2.0
```

**Comment by adbeda** [comment]
@conda-forge/help-python will you please add this recipe to conda-forge? I'm working on another package that depends on this.
Thanks!

**Comment by agcopenhaver** [comment]
@rluria14 please list me or @adbeda as a maintainer as well

**Comment by adbeda** [comment]
@marcelotrevisani I w'd appreciate if make check this one out too

File: recipes/cfn-flip/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  entry_points:
+    - cfn-flip=cfn_flip.main:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - click
+    - pip
+    - python
+    - pyyaml >=4.1
```
Comment (scopatz) [inline recipes/cfn-flip/meta.yaml]: Are click and pyyaml really needed to install the package?

Comment (scopatz) [reply recipes/cfn-flip/meta.yaml]: It seems like they are only run deps

**Comment by adbeda** [comment]
I agree to be a co-maintainer. thanks!

File: recipes/cfn-flip/meta.yaml (hunk 2 above)
Comment (rluria14) [reply recipes/cfn-flip/meta.yaml]: thank you. changes have been made to have click and pyyaml only as run deps.

**Pull request merged**